### PR TITLE
Added setup script to setup systemd service on ubuntu

### PR DESCRIPTION
**Pull request by ifel**
Created a setup script which sets up a systemd service instead of a cronjob.

1. Split basic setup and actual setup to simplify what a user should do + have most of the commands in the repo, so user section will remain the same while actual setup may be easily modified
2. Systemd service writes logs with timestamps + does log rotation.
3. Service restart with pulling a new version of code from the repo every 2 hours and not on reboot.
4. If the service dies (exception, etc), systemd will automatically restart it.
5. Native for modern Linux

Tests:
1. It works:
```
Mar 20 19:50:52 ip-172-31-9-159 systemd[1]: Started uaswrod service.                                                                                                                                      -- Subject: A start job for unit uasword.service has finished successfully
-- Defined-By: systemd                                                                                                                                                                                    -- Support: http://www.ubuntu.com/support
--                                                                                                                                                                                                        -- A start job for unit uasword.service has finished successfully.
--                                                                                                                                                                                                        -- The job identifier is 3355.                                                                                                                                                                            Mar 20 19:50:52 ip-172-31-9-159 uasword_start.sh[17955]: Already up to date.
Mar 20 19:50:52 ip-172-31-9-159 uasword_start.sh[17956]: Current branch master is up to date.                                                                                                             Mar 20 19:50:56 ip-172-31-9-159 uasword_start.sh[17958]: added 329 packages, and audited 330 packages in 3s
Mar 20 19:50:56 ip-172-31-9-159 uasword_start.sh[17958]: 30 packages are looking for funding                                                                                                              Mar 20 19:50:56 ip-172-31-9-159 uasword_start.sh[17958]:   run `npm fund` for details
Mar 20 19:50:56 ip-172-31-9-159 uasword_start.sh[17958]: found 0 vulnerabilities                                                                                                                          Mar 20 19:50:58 ip-172-31-9-159 uasword_start.sh[17992]: Installing Ubuntu dependencies...
Mar 20 19:50:58 ip-172-31-9-159 uasword_start.sh[18000]: Hit:1 http://eu-central-1.ec2.archive.ubuntu.com/ubuntu focal InRelease
Mar 20 19:50:58 ip-172-31-9-159 uasword_start.sh[18000]: Hit:2 http://eu-central-1.ec2.archive.ubuntu.com/ubuntu focal-updates InRelease
Mar 20 19:50:58 ip-172-31-9-159 uasword_start.sh[18000]: Hit:3 http://eu-central-1.ec2.archive.ubuntu.com/ubuntu focal-backports InRelease
Mar 20 19:50:58 ip-172-31-9-159 uasword_start.sh[18000]: Hit:4 http://ppa.launchpad.net/jgmath2000/et/ubuntu focal InRelease
Mar 20 19:50:58 ip-172-31-9-159 uasword_start.sh[18000]: Hit:5 https://deb.nodesource.com/node_16.x focal InRelease
Mar 20 19:50:58 ip-172-31-9-159 uasword_start.sh[18000]: Hit:6 http://security.ubuntu.com/ubuntu focal-security InRelease
Mar 20 19:50:59 ip-172-31-9-159 uasword_start.sh[18000]: Reading package lists...
Mar 20 19:51:00 ip-172-31-9-159 uasword_start.sh[18367]: Reading package lists...
Mar 20 19:51:00 ip-172-31-9-159 uasword_start.sh[18367]: Building dependency tree...
Mar 20 19:51:00 ip-172-31-9-159 uasword_start.sh[18367]: Reading state information...
...
Mar 20 19:51:25 ip-172-31-9-159 uasword_start.sh[17992]: Starting process for https://garant-logistika.ru/
Mar 20 19:51:26 ip-172-31-9-159 uasword_start.sh[17992]: Starting process for https://www.flagmanamur.ru/
Mar 20 19:51:27 ip-172-31-9-159 uasword_start.sh[17992]: Starting process for https://tranzit-auto.ru/
Mar 20 19:51:28 ip-172-31-9-159 uasword_start.sh[17992]: Starting process for http://sztcom.ru/
Mar 20 19:51:29 ip-172-31-9-159 uasword_start.sh[17992]: Starting process for http://magistrans.ru/
Mar 20 19:51:59 ip-172-31-9-159 uasword_start.sh[17992]: http://sztcom.ru/ is not reachable. Retrying in 60000 ms...
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: ┌─────────┬──────────────────────────────────────────────────────────────┬──────────┬──────────┬───────┐
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: │ (index) │                             url                              │ Requests │ Errors,% │ Req/s │
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: ├─────────┼──────────────────────────────────────────────────────────────┼──────────┼──────────┼───────┤
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: │    0    │ 'https://tp.media/r?marker=48286&trace_id=Zz62dfe72b5c0a...' │  11757   │    0     │  183  │
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: │    1    │ 'https://airport-rzhevka.ru/obzor-poiskovyx-sistem-aviab...' │   1167   │    0     │  23   │
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: │    2    │                      'https://pgk.ru/'                       │    68    │    0     │   0   │
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: │    3    │                   'https://www.povoz.ru/'                    │    18    │    0     │   0   │
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: │    4    │                    'https://oooyata.ru/'                     │    14    │    0     │   0   │
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: │    5    │                   'http://magistrans.ru/'                    │    0     │    0     │   0   │
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: └─────────┴──────────────────────────────────────────────────────────────┴──────────┴──────────┴───────┘
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: Total Requests 13052 | Active runners 6 of 7 | Total rps 206
Mar 20 19:52:30 ip-172-31-9-159 uasword_start.sh[17992]: https://tranzit-auto.ru/ is not reachable. Retrying in 60000 ms...
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: ┌─────────┬──────────────────────────────────────────────────────────────┬──────────┬──────────┬───────┐
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │ (index) │                             url                              │ Requests │ Errors,% │ Req/s │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: ├─────────┼──────────────────────────────────────────────────────────────┼──────────┼──────────┼───────┤
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │    0    │ 'https://tp.media/r?marker=48286&trace_id=Zz62dfe72b5c0a...' │  26449   │    0     │  265  │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │    1    │ 'https://airport-rzhevka.ru/obzor-poiskovyx-sistem-aviab...' │   7019   │    0     │  100  │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │    2    │                      'https://pgk.ru/'                       │   128    │    0     │   0   │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │    3    │                   'https://www.povoz.ru/'                    │    38    │    0     │   0   │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │    4    │                    'https://oooyata.ru/'                     │    34    │    0     │   0   │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │    5    │                   'http://magistrans.ru/'                    │    22    │    0     │   0   │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: │    6    │                'https://www.flagmanamur.ru/'                 │    10    │    0     │   0   │
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: └─────────┴──────────────────────────────────────────────────────────────┴──────────┴──────────┴───────┘
Mar 20 19:53:30 ip-172-31-9-159 uasword_start.sh[17992]: Total Requests 33756 | Active runners 7 of 9 | Total rps 365
```

2. It restarts and pulls a new code from git:
```
Mar 20 21:31:30 ip-172-31-32-202 systemd[1]: uasword.service: Service reached runtime time limit. Stopping.                                                                                                                       [58/1061]
Mar 20 21:31:31 ip-172-31-32-202 uasword_start.sh[12082]: https://www.fesco.ru/ is not reachable. Retrying in 60000 ms...
Mar 20 21:31:32 ip-172-31-32-202 uasword_start.sh[12082]: http://sztcom.ru/ is not reachable. Retrying in 60000 ms...
Mar 20 21:31:36 ip-172-31-32-202 uasword_start.sh[12082]: https://www.skif-cargo.ru/ is not reachable. Retrying in 60000 ms...
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: State 'stop-final-sigterm' timed out. Killing.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12082 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12083 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12084 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12085 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12086 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12087 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12088 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12507 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12508 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12509 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Killing process 12510 (node) with signal SIGKILL.
Mar 20 21:31:40 ip-172-31-32-202 systemd[1]: uasword.service: Failed with result 'timeout'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- The unit uasword.service has entered the 'failed' state with result 'timeout'.
Mar 20 21:31:42 ip-172-31-32-202 systemd[1]: uasword.service: Scheduled restart job, restart counter is at 1.
-- Subject: Automatic restarting of a unit has been scheduled
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- Automatic restarting of the unit uasword.service has been scheduled, as the result for
-- the configured Restart= setting for the unit.
Mar 20 21:31:42 ip-172-31-32-202 systemd[1]: Stopped uaswrod service.
-- Subject: A stop job for unit uasword.service has finished
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- A stop job for unit uasword.service has finished.
--
-- The job identifier is 2009 and the job result is done.
Mar 20 21:31:42 ip-172-31-32-202 systemd[1]: Starting uaswrod service...
-- Subject: A start job for unit uasword.service has begun execution
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- A start job for unit uasword.service has begun execution.
--
-- The job identifier is 2009.
Mar 20 21:31:42 ip-172-31-32-202 systemd[1]: Started uaswrod service.
-- Subject: A start job for unit uasword.service has finished successfully
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
--
-- A start job for unit uasword.service has finished successfully.
--
-- The job identifier is 2009.
Mar 20 21:31:42 ip-172-31-32-202 uasword_start.sh[13545]: Already up to date.
Mar 20 21:31:42 ip-172-31-32-202 uasword_start.sh[13546]: Current branch master is up to date.
Mar 20 21:31:43 ip-172-31-32-202 uasword_start.sh[13548]: up to date, audited 330 packages in 1s
```

### CODE REVIEW

**Review by mgrybyk — APPROVED** [review]
👍

**Comment by mgrybyk** [comment]
@ifel can I merge the PR?

**Comment by ifel** [comment]
> @ifel can I merge the PR?

Yes, please. You're the owner of the target repo, and have write access to it. I cannot merge it myself ("Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests")

**Pull request merged**
